src/components/ContactForm/index.jsx

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable react/no-unknown-property */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React, { useState } from "react";
+import "./index.css";
+import { Row, Col } from "react-bootstrap";
+import { useTranslation } from "react-i18next";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faYoutube,
+  faFacebook,
+  faTwitter,
+  faInstagram,
+} from "@fortawesome/free-brands-svg-icons";
+import ContactUsChannels from "./ContactUsChannels.svg";
+import Piggy from "./Piggy.svg";
```

Comment:
Imgs names doesn’t need to be capitalized, and it would be better if you indicate that the variable is an img like naming it piggyImg

Reply:
PTAL